server.js

```diff
@@ -12,6 +13,7 @@ var LocationTrend = require('./mongodb/models/locationTrend.js');
 Local Dependencies
 **************************************************************/
 var countryTrends = require('./utilities/twitterAPI/countryTrends');
+var cityTrends = require('./utilities/twitterAPI/cityTrends');)
 var usRoutes = require('./utilities/middleware/usRoutes');
```

Comment:
@SimonDing87 @kcvan found some extra semicolons on line 16, not sure if the app will ignore that or crash, but we should probably remove it :)